client/packages/react/src/InstantReactAbstractDatabase.tsx

```diff
@@ -261,6 +261,14 @@ export default abstract class InstantReactAbstractDatabase<
     return state;
   };
 
+  useUser = () => {
```

Comment:
nit: I would add a docblock on the function with an example. This way when a user hovers over it in intellisense, they can see how to use it right away